Problem of compatibility with Java Versions

Hi. I'm getting a problem of compatibility with Java Versions. My project Java version is 8 and the _gradle-minify-plugin_ apparently needs Java 11. Is there an workaround or the only solution is to upgrade Java version? 

The erros is:

> A problem occurred configuring project ':app'.
> > Could not resolve all artifacts for configuration ':app:classpath'.
>    > Could not resolve org.gradle-webtools.minify:gradle-minify-plugin:1.2.1.
>      Required by:
>          project :app > org.gradlewebtools.minify:org.gradlewebtools.minify.gradle.plugin:1.2.1
>       > No matching variant of org.gradle-webtools.minify:gradle-minify-plugin:1.2.1 was found. The consumer was configured to find a runtime of a library compatible with Java 8, packaged as a jar, and its dependencies declared externally but:
>           - Variant 'apiElements' capability org.gradle-webtools.minify:gradle-minify-plugin:1.2.1 declares a library, packaged as a jar, and its dependencies declared externally:
>               - Incompatible because this component declares an API of a component compatible with Java 11 and the consumer needed a runtime of a component compatible with Java 8
>           - Variant 'runtimeElements' capability org.gradle-webtools.minify:gradle-minify-plugin:1.2.1 declares a runtime of a library, packaged as a jar, and its dependencies declared externally:
>               - Incompatible because this component declares a component compatible with Java 11 and the consumer needed a component compatible with Java 8
> 
> * Try:
> Run with --info or --debug option to get more log output. Run with --scan to get full insights.
> 
> * Exception is:
> org.gradle.api.ProjectConfigurationException: A problem occurred configuring project ':app'.
> 
> Caused by: org.gradle.api.internal.artifacts.ivyservice.DefaultLenientConfiguration$ArtifactResolveException: Could not resolve all artifacts for configuration ':app:classpath'.
> 
> Caused by: org.gradle.internal.resolve.ModuleVersionResolveException: Could not resolve org.gradle-webtools.minify:gradle-minify-plugin:1.2.1.
> Required by:
>     project :app > org.gradlewebtools.minify:org.gradlewebtools.minify.gradle.plugin:1.2.1
> Caused by: org.gradle.internal.component.NoMatchingConfigurationSelectionException: No matching variant of org.gradle-webtools.minify:gradle-minify-plugin:1.2.1 was found. The consumer was configured to find a runtime of a library compatible with Java 8, packaged as a jar, and its dependencies declared externally but:
>   - Variant 'apiElements' capability org.gradle-webtools.minify:gradle-minify-plugin:1.2.1 declares a library, packaged as a jar, and its dependencies declared externally:
>       - Incompatible because this component declares an API of a component compatible with Java 11 and the consumer needed a runtime of a component compatible with Java 8
>   - Variant 'runtimeElements' capability org.gradle-webtools.minify:gradle-minify-plugin:1.2.1 declares a runtime of a library, packaged as a jar, and its dependencies declared externally:
>       - Incompatible because this component declares a component compatible with Java 11 and the consumer needed a component compatible with Java 8

--- a/.github/workflows/develop.yml
+++ b/.github/workflows/develop.yml
@@ -16,6 +16,12 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+
       - name: build
         run: ./gradlew clean assemble
 

--- a/.github/workflows/feature.yml
+++ b/.github/workflows/feature.yml
@@ -17,6 +17,12 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+
       - name: build
         run: ./gradlew clean assemble
 

--- a/.github/workflows/master.yml
+++ b/.github/workflows/master.yml
@@ -16,6 +16,12 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+
       - name: build
         run: ./gradlew clean assemble
 
@@ -27,6 +33,15 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
         run: ./gradlew sonarqube
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+
+      - name: build
+        run: ./gradlew clean assemble
 
       - name: release
         env:

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -9,7 +9,7 @@ plugins {
 }
 
 group = "org.gradle-webtools.minify"
-version = "1.2.2"
+version = "1.2.3"
 
 tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
     kotlinOptions.jvmTarget = "1.8"
@@ -52,7 +52,6 @@ dependencies {
     implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.2.1")
     implementation("com.google.javascript:closure-compiler:v20210106")
     implementation("org.padler:closure-stylesheets:1.6.2")
-    implementation("org.jetbrains.kotlin:kotlin-reflect:1.5.20")
 
     testImplementation("io.kotest:kotest-runner-junit5:4.6.0")
     testImplementation("io.kotest:kotest-assertions-core:4.6.0")

